packages/react/src/components/Toggletip/index.tsx

```diff
@@ -232,7 +232,7 @@ Toggletip.propTypes = {
   ]),
 
   /**
-   * Provide an offset value for alignment axis.
+   * Provide an offset value for alignment axis. Only takes effect when `autoalign` is enabled.
```

Comment:
This should've been marked experimental to match `autoAlign`

```suggestion
   * **Experimental:** Provide an offset value for alignment axis. Only takes effect when `autoalign` is enabled.
```